### PR TITLE
Add link for deployment cancel action

### PR DESCRIPTION
**Pull request by Gerg**
Only visible if the deployment is cancelable

### CODE REVIEW

File: app/actions/deployment_cancel.rb [hunk 1]
```diff
@@ -30,9 +30,7 @@ def cancel(deployment:, user_audit_info:)
       private
 
       def valid_state?(deployment)
```
Comment (selzoc) [inline app/actions/deployment_cancel.rb]: I feel like i would just get rid of this method if we're delegating to the `DeploymentModel`

Comment (sethboyles) [reply app/actions/deployment_cancel.rb]: Agreed.  Especially if the parameter here isn't being checked.

Comment (sethboyles) [reply app/actions/deployment_cancel.rb]: ^^

**Comment by selzoc** [comment]
Going to close/reopen this to trigger our new checks

**Pull request merged**